tests/test_layers.py

```diff
@@ -993,6 +993,19 @@ def model_func(wrapped_import, inputs: torch.Tensor):
   verify_torch_and_convert_to_returnn(model_func, inputs=x)
 
 
+def test_transpose_unsqueeze():
```

Comment:
This test is for `Unflatten`? How?

Reply:
Ah sorry, I thought of sth else. `Unflatten` is used by `reshape`, and thus here in `unsqueeze`.